### PR TITLE
fix: `Task::Role#exists?` always return true.

### PR DESCRIPTION
**Pull request by hasimo**
I got  a error of `Aws::IAM::Errors::NoSuchEntity: Role not found for ecsEventsRole` during creating the scheduled tasks. 
It looks like `Task::Role#exists?` always return `true`. Because  `Aws::IAM::Resource#role` always return `AWS::IAM::Role` object.

### CODE REVIEW

File: spec/tasks/role_spec.rb [hunk 1]
```diff
@@ -51,6 +51,15 @@
         allow(resource).to receive(:role).with(ElasticWhenever::Task::Role::NAME).and_return(nil)
       end
 
+      it "returns false" do
```
Comment (wata727) [inline spec/tasks/role_spec.rb]: I think this test case is incorrect. Please remove this one :pray:

File: spec/tasks/role_spec.rb [hunk 2]
```diff
@@ -51,6 +51,15 @@
...
+        expect(ElasticWhenever::Task::Role.new(option)).not_to be_exists
+      end
+    end
+    context "when role not found (deloay load)" do
```
Comment (wata727) [inline spec/tasks/role_spec.rb]: Maybe `deloay load` is not needed if you remove the above test case.

File: spec/tasks/role_spec.rb (hunk 1 above)
Comment (hasimo) [reply spec/tasks/role_spec.rb]: done.

**Comment by wata727** [comment]
:+1:

Released v0.3.2
https://rubygems.org/gems/elastic_whenever/versions/0.3.2

**Pull request merged**